src/website/typography/Heading/Heading.js

```diff
@@ -0,0 +1,54 @@
+export default function Heading({
+    typeStyle = "heading-lg",
+    level = 'h2',
+    textColor = 'text-primary-800',
+    className,
+    children
+}) {
+    let classNames = ''
+    const Tag = `${level}`;
```

Comment:
```suggestion
    const Tag = level;
```

But we should check we only get a valid value here and not some random string.